src/objects/LOD.js

```diff
@@ -32,6 +32,8 @@ LOD.prototype = Object.assign( Object.create( Object3D.prototype ), {
 	constructor: LOD,
 
 	isLOD: true,
+	
+	currentLevel: 0,
```

Comment:
Um, I think  I would favor `LOD.getCurrentLevel()`.  Users could think this property is writable.

Reply:
I see your point. Is there a performance concern of a getter over a direct access property, or are we beyond that in three.js?

I.e.
`var _currentLevel`
and
`Object.defineProperties ... get: function() { return _currentLevel; }`

Reply:
Just make a private variable and then a simple get method. I don't think `defineProperties` is necessary here.

Reply:
Ok, done. Thanks for your input